CRD-DTR/DrugHasREMS/R4/resources/Questionnaire-R4-Prescriber-Enrollment.json

```diff
@@ -0,0 +1,337 @@
+                            "type": "string",
+                            "required": false
+                        },
+                        {
+                            "linkId": "2.3.5",
+                            "text": "Email",
+                            "type": "string",
+                            "required": false
+                        }
+                    ]
+                },
+                {
+                    "linkId": "2.4",
+                    "type": "group",
+                    "text": "Office Contact #2 (Optional)",
+                    "item": [
+                        {
+                            "linkId": "2.4.1",
+                            "text": "First Name",
+                            "type": "string",
+                            "required": false
+                        },
+                        {
+                            "linkId": "2.4.2",
+                            "text": "Last Name",
+                            "type": "string",
+                            "required": false
+                        },
+                        {
+                            "linkId": "2.4.3",
+                            "text": "Office Phone Number",
+                            "type": "string",
+                            "required": false
+                        },
+                        {
+                            "linkId": "2.4.4",
+                            "text": "Office Fax Number",
+                            "type": "string",
+                            "required": false
+                        },
+                        {
+                            "linkId": "2.4.5",
+                            "text": "Email",
+                            "type": "string",
+                            "required": false
+                        }
+                    ]
+                },
+                {
+                    "linkId": "2.5",
+                    "text": "Office Contacts can be updated by visiting www.turaliorems.com or contacting the TURALIO REMS Coordinating Center at 1-833-TURALIO (833-887-2546).",
+                    "type": "display"
+                }
+            ]
+        },
+        {
+            "linkId": "3",
+            "type": "group",
+            "text": "Prescriber Attestations",
+            "item": [
+                {
+                    "linkId": "3.1",
+                    "text": "NEED TEXT", 
+                            
+                    "type": "display",
+                    "readOnly": true
+                  },
+                  {
+                    "linkId": "3.2",
+                    "extension": [
+                      {
+                        "url": "http://hl7.org/fhir/StructureDefinition/sub-questionnaire",
+                        "valueCanonical": "questionnaire/provider-signature"
+                      }
+                    ],
+                    "type": "display"
+                  }
```

Comment:
more specifically, the extension containing the `questionnaire/provider-signature` indicates that the sub-questionnaire for provider signature will be substituted in here, and it will include other relevant fields for recording the signature.